hassio-google-drive-backup/backup/static/layouts/partials/error-messages.jinja2

```diff
@@ -0,0 +1,691 @@
+{% import "layouts/macros.jinja2" as macros %}
```

Comment:
> I'll need to do a little more testing of these before I'm confidant everything works correctly, but at a first pass everything looks good for submission.

Great idea! While I double checked everything, it is a good idea to do more testing on that.
I've found out that some errors were using the wrong id or duplicated ids, so I fixed and tested those.
Also in the errorMessage macro, I put the error id in both the `id` and `class` fields, since some of the code was calling them using their id and other part using their name in the class field:
```html
<div class="card default-hidden error_card {{ id }}" id="{{ id }}">
```